ui/app/src/components/GlobalConfigManagement/GlobalConfigManagement.tsx

```diff
@@ -89,7 +89,8 @@ export default function GlobalConfigManagement() {
     if (type === 'replace') {
       setContent(sample);
     } else {
-      setContent(content + sample);
+      const currentContent = aceEditorRef.current.getValue();
```

Comment:
@rart the issue was that if you edit the editor the content is not set on each update (I guess that it's to avoid calling n times setContent while typing). So, I'm getting the content that will be append to the sample directly from the editor value